test/pages/SingleProduct.test.tsx

```diff
@@ -1,3 +1,162 @@
+// import { render, screen } from '@testing-library/react';
```

Comment:
I think you don't need this commented lines, you can remove them. Then we're good on this PR